build-support/bin/native/bootstrap.sh

```diff
@@ -62,6 +62,11 @@ function ensure_build_prerequisites() {
     sh ${rustup} -y --no-modify-path 1>&2
     rm -f ${rustup}
     ${RUSTUP_HOME}/bin/rustup override set stable 1>&2
+
```

Comment:
I agree that this is a portability bit, and should probably go over there. Perhaps in a second pass though, we can use musl by default on linux, such that that is what is actually used in travis?